app/about/page.tsx

```
@@ -0,0 +1,21 @@
+import AboutBanner from "@/app/components/about/AboutBanner";
+import OurStory from "@/app/components/about/OurStory";
+import OurMission from "@/app/components/about/OurMission";
+import OurTeam from "@/app/components/about/OurTeam";
```

ESLint

[error] 'OurTeam' is defined but never used.

(@typescript-eslint/no-unused-vars)